provision/aws/aws_add_secret_to_openshift_env_file.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+source ./.env
+
+KEYCLOAK_MASTER_PASSWORD_SECRET_NAME=${KEYCLOAK_MASTER_PASSWORD_SECRET_NAME:-"keycloak-master-password"}
+# Force eu-central-1 region for secrets manager so we all work with the same secret
+REGION="eu-central-1"
+KEYCLOAK_MASTER_PASSWORD=$(aws secretsmanager get-secret-value --region $REGION --secret-id "$KEYCLOAK_MASTER_PASSWORD_SECRET_NAME" --query SecretString --output text --no-cli-pager)
+
+echo "KC_ADMIN_PASSWORD='$KEYCLOAK_MASTER_PASSWORD'" >> ../openshift/.env
```

Comment:
I see two problems with this script's execution,

1. This would create duplicates when run multiple times.
2. And as a User might not know necessarily which is the older password once I make an update.

My suggestion would be to make it more disambiguous., such as overwriting the password in the .env for the KC_ADMIN_PASSWORD rather than appending it.

@tkyjovsk and @ahus1 any thoughts ?

Reply:
Agreed, it shouldn't be appended on every run. 

@mhajas - could we make it a default like we did it for `KC_NAMESPACE_PREFIX` with `default "$(...)"`  so it wouldn't need to be in the `.env` file at all?

Reply:
@kami619 @ahus1 Thank you for the feedback. I didn't like the idea of having openshift Taskfile dependent on `aws` command, however, using `|| echo admin` should work if `aws` command is not present on the machine, so I guess this is much more user-friendly. Please take a look at the latest changes.

Reply:
thanks for addressing that @mhajas . I will try it out and merge it 👍🏼